recipes/systemd/meta.yaml

```
@@ -0,0 +1,136 @@
+{% set name = "systemd" %}
+{% set version = "243" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/systemd/systemd/archive/v{{ version }}.tar.gz
+  sha256: 0611843c2407f8b125b1b7cb93533bdebd4ccf91c99dffa64ec61556a258c7d1
+  patches:
+    # fixes compilation issue for missing socket decl
+    # from https://bugs.freedesktop.org/show_bug.cgi?id=67075
+    - l2tp.h.patch
+    - netlink.h.patch
+    # adds hook to secure_getenv, if needed
+    - env-util.patch
+    - proc-cmdline.c.patch
+    # adds signal header to get sigemptyset declaration
+    - copy.patch
+    # adds missing definitions
+    - missing_fs.patch
+    - missing_magic.h.patch
+    - missing_prctl.h.patch
+    - missing_fcntl.h.patch
+    - missing_socket.h.patch
+    # adds missing syncfs() implementation
+    - fs-util.c.patch
+    # don't confuse AF_ALG on systems where it is not availble
+    - khash.c.patch
+    # adds missing time definition hooks
+    - time-util.h.patch
+    - time-util.c.patch
+    - sd-event.c.patch
+    # adds missing ioctls definition hooks
+    - terminal-util.c.patch
+    # Adds missing neighbour.h enums and defines
+    - networkd-fdb.h.patch
+    - netlink-types.c.patch
+    # add missing BPF_XOR defines
+    - arp-util.c.patch
+    - dhcp-network.c.patch
+    # add missing K_OFF def
+    - logind-session.c.patch
+    # add missing O_PATH def
+    - btrfs-util.c.patch
+    - chown-recursive.c.patch
+    # add missing types.h
+    - bpf.h.patch
+    # Add missing loop info
+    - loop-util.c.patch
+    - dissect-image.c.patch
+    # adds missing video defs
+    - v4l_id.c.patch
+    # add missing inputs
+    - udev-builtin-input_id.c.patch
+    # fixes for prctl.h
+    - nspawn-oci.c.patch
+    # avoid missing statfs.f_flags
+    - nspawn-patch-uid.c.patch
+    # add missing syscall
+    - machine-dbus.c.patch
+    # add missing capabilities defines
+    - missing_capability.h.patch
+    # missing in.h defines
+    - resolved-dns-scope.c.patch
+    - resolved-dns-stream.c.patch
+    # add missing fcntl.h
+    - socket-proxyd.c.patch
+    - coredump-vacuum.c.patch
+    # avoid clock_adjtime
+    - timesyncd-manager.c.patch
+    # avoid prlimit
+    - nspawn.c.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
```

Using `python >=3.5` isn't going to work for constraining the version. I guess it should be removed in favour of `skip`?
```suggestion
  skip: true  # [py<35 or not linux]
```

---

Why @chrisburr? This is how we ship `noarch: python` all the time.

---

Asking out of curiosity more than anything else. I missed a bunch of stuff in conda 101. :p

---

If it's `noarch: xxxx` there will be a single build on Linux that can be installed on any platform. I think in it's current form[1] conda-smithy will generate a Python 2.7, Python 3.6 and Python 3.7 build as there are no `noarch` specifiers.

[1] Assuming I haven't missed any config files that override things

---

Ah interesting. I thought conda smithy was smart enough to not make CI jobs for those and then fix the python version in the conda build config locally